cmd/collection/main.go

```diff
@@ -339,12 +340,16 @@ func main() {
 				syncFactory,
 			)
 
+			heightEvents := gadgets.NewHeights()
+			node.ProtocolEvents.AddConsumer(heightEvents)
```

Comment:
is this what I should be subscribe to as well to be notified about epochs at the networking layer?

Reply:
Yes, `node.ProtocolEvents` is what you want to use 👍

Reply:
sweet